docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>The <code># mlrun: start-code</code> and <code># mlrun: end-code</code> annotations can be used to convert different code sections to different MLRun functions in the same notebook. &nbsp;To do so add the name of the MLRun function to the end of the annotation as shown in the example below.</p><p><strong><em>(move the note to after the example)</em></strong></p><p> <strong>Note:</strong></p><p> Make sure to use the name given to the <code>code_to_function</code> parameter (name='my-function-name' in the example above) so that all relevant <code>start-code</code> and <code>end-code</code> annotations will be included. If none of the annotations are marked with a function name, all of annotations without any name will be used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>